Jenkinsfile

```diff
@@ -1,21 +1,61 @@
 elifeLibrary {
+    def isNew
+    def candidateVersion
     def commit
+
     stage 'Checkout', {
         checkout scm
         commit = elifeGitRevision()
     }
 
-    stage 'Build image', {
-        sh 'docker build -t elife/sciencebeam-alignment .'
-    }
+    node('containers-jenkins-plugin') {
+        stage 'Build images', {
+            checkout scm
+            dockerComposeBuild(commit)
+            try {
```

Comment:
we should extract this as `dockerComposeRun` similar to https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/dockerComposeBuild.groovy